scripts/Dockerfile

```diff
@@ -81,7 +81,7 @@ FROM output-base as output-next
 COPY --from=builder /build/${APP_HOME}/next.config.js /webapp
 COPY --from=builder /build/${APP_HOME}/next-modules/ /webapp/next-modules/
 COPY --from=builder /build/${APP_HOME}/public /webapp/public
-COPY --from=builder /build/${APP_DIST_HOME} /webapp/.next
+COPY --from=builder /build/${APP_DIST_HOME}/.next /webapp/.next
```

Comment:
wtf, how did this work until now?!?

Reply:
It's a change in nx10, they now move a package.json, public and next.config.js to the dist folder, and move .next one level down.